docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -203,6 +203,8 @@ public class CounterBean {
 [[supported_features]]
 == Supported Features
 
+CDI Lite is supported. That includes:
```

Comment:
The list below includes features that aren't Lite only such as decorators.

Reply:
I'll probably just remove these sentences that I added (here in _Supported Features_ and in _Limitations_), as it's hard to put succinctly.